src/actions/vstudio/vs2005_solution.lua

```diff
@@ -32,7 +32,7 @@
 
 	function sln2005.generate(wks)
 		-- Mark the file as Unicode
-		_p('\239\187\191')
+		premake.utf8()
```

Comment:
This outputs the BOM without a trailing newline, but projects generated manually with VS have that newline. This could be fixed a few ways:

```lua
p.utf8ln()   -- i.e. UTF8+newline; would have to be written

p.utf8(); p.outln()  -- I think I probably like this one best

p.w('\239\187\191')
```